numpy/lib/function_base.py

```diff
@@ -338,6 +338,10 @@ def histogram(a, bins=10, range=None, normed=False, weights=None,
         if (mn > mx):
             raise AttributeError(
                 'max must be larger than min in range parameter.')
+        if not np.all(np.isfinite([mn, mx])):
+            raise AttributeError(
```

Comment:
Not really, can be put in this one if you want too. But you may have to go fixing failing tests after the change, if it turns into too much work, we can simply open an issue and come back to it later.